prover/src/lib.rs

```diff
@@ -517,7 +518,10 @@ pub trait Prover {
         // evaluate composition polynomial columns over the LDE domain
         #[cfg(feature = "std")]
         let now = Instant::now();
```

Comment:
Hey @irakliyk. What do you think of removing build_constraint_commitment and just using build_trace_commitment for it instead?